checkfavors.lic

```diff
@@ -53,6 +56,41 @@ class CheckFavors
     when 'your sacrifice is properly prepared'
       true
     end
+    fput("stow my orb")
+    exit
```

Comment:
Won't this always exit the script here instead of returning a boolean like the function name implies? If so, I think this will break anything relying on the return value, e.g. https://github.com/rpherbig/dr-scripts/blob/b2398fb2ffa8a530b24483181b22fa3ca49d01ea/checkfavors.lic#L31

Reply:
Ahh, okies.  Still kinda new so I'll poke around with it!  :D